### PR TITLE
Fix a few bugs, and make a few clarifications in the android quick start guide.

### PR DESCRIPTION
**Pull request by JayNewstrom**
## Description:
- **What's changed?** Fix a few bugs, and make a few clarifications in the android quick start guide.
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-XXXXXX](https://oktainc.atlassian.net/browse/OKTA-XXXXXX)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/android/configmid.md [hunk 1]
```diff
@@ -27,4 +27,14 @@
     }
     ```
 
+1. Add the following code to the dependencies block of the app's build file, `app/build.gradle`:
+
+    ```groovy
+    dependencies {
+        // Used from Okta Mobile SDK code snippets.
```
Comment (mauricesharp-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/android/configmid.md:34]:
```suggestion
        // These packages are used in the Okta Mobile SDK code snippets.
```

File: packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/android/createproject.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
+> **Note:** The guide requires Android Studio (Arctic Chipmunk 2021.2.1)
+
 1. Create a new Kotlin Android project called `browser_sign_in` with an **Empty Activity** in Android Studio. For information on creating a project, see the [Android Studio documentation](https://developer.android.com/training/basics/firstapp/creating-project).
 
+    - Name: `browser_sign_in`
+    - Package name: `com.okta.android.samples.browser_sign_in`
+    - Language: `Kotlin`
+    - Minimum SDK: `API 23: Android 6.0 (Marshmallow)`
+
 2. Add the following code to the `app/build.gradle` to enable Android features used by the Okta SDK that aren't available on older Android versions:
```
Comment (mauricesharp-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/android/createproject.md:10]: Do these add features that are after Android 6.0? Are they required or optional features, that is, must these packages load for an app to run?

This example assumes the packages are required... it requires the minimum Android version

> 2. Add the following code to the `app/build.gradle` file to enable features used by the Okta SDK that require Android xx.xx or later:

**Review by mauricesharp-okta — COMMENTED** [review]
A couple of nits.

File: packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/android/createproject.md (hunk 2 above)
Comment (JayNewstrom) [reply packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/android/createproject.md:10]: These are required for the app to run.

**Review by robertnicolo-okta — APPROVED** [review]
lgtm!

File: packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/android/createproject.md (hunk 2 above)
Comment (mauricesharp-okta) [reply packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/android/createproject.md:10]: The important thing is to be specific about what versions of Android: "older Android versions" doesn't tell them. They need to know what versions of Android are supported, which appears to be on line 8. So if this line is loading packages that are Android 6 or later, then the line could be (the fact that it's Android 6.0 or later is taken care of above... if the packages require later versions of Android, then the language needs to change to indicate that.):
```suggestion
2. Add the following to the `app/build.gradle` to enable features required by the Okta Android SDK:
```

**Comment by JayNewstrom** [comment]
@mauricesharp-okta please post review, and I'll follow up if you have more comments. Thanks!

**Pull request merged**